blocks/bio-detail/bio-detail.js

```diff
@@ -19,7 +19,11 @@ function createSocialMediaLink(linkName, className, iconPath) {
     const linkImage = img({ class: className });
     linkImage.src = `${window.hlx.codeBasePath}/icons/${iconPath}`;
     anchor.appendChild(linkImage);
-    return div({ class: 'social-media-link' }, anchor);
+    const socialMediaLink = div({ class: 'social-media-link' }, anchor);
+    socialMediaLink.addEventListener('click', () => {
+      window.location.href = linkName.textContent;
```

Comment:
current link clicks are going to wrong location.. is this authoring issue ? laso do we need this line code again if a link already has correct link in it's href

Reply:
links were random. changed the links now. Pls check